doc/python/decor.rst

```diff
@@ -0,0 +1,246 @@
+
+        Apply a mechanism with a region using the name of the mechanism.
+        The mechanism will use the parameter values set in the mechanism catalogue.
+        Returns a unique identifier that can be used to query the local indexes (see :gen:`index`) assigned to the placed items on the cable cell.
+
+        :param str region: description of the region.
+        :param str mechanism: the name of the mechanism.
+
+    .. method:: place(locations, const arb::mechanism_desc& d)
+
+        Place one instance of synapse described by ``mechanism`` to each location in ``locations``.
+        Returns a unique identifier that can be used to query the local indexes (see :gen:`index`) assigned to the placed items on the cable cell.
+
+        :param str locations: description of the locset.
+        :param str mechanism: the name of the mechanism.
+        :rtype: int
+
+    .. method:: place(locations, mechanism)
+        :noindex:
+
+        Place one instance of synapse described by ``mechanism`` to each location in ``locations``.
+        Returns a unique identifier that can be used to query the local indexes (see :gen:`index`) assigned to the placed items on the cable cell.
+
+        :param str locations: description of the locset.
+        :param mechanism: the mechanism.
+        :type mechanism: :py:class:`mechanism`
+        :rtype: int
+
+    .. method:: place(locations, site)
+        :noindex:
+
+        Place one gap junction site at each location in ``locations``.
+        Returns a unique identifier that can be used to query the local indexes (see :gen:`index`) assigned to the placed items on the cable cell.
+
+        :param str locations: description of the locset.
+        :param site: indicates a gap junction site..
+        :type site: :py:class:`gap_junction_site`
+        :rtype: int
+
+    .. method:: place(locations, stim)
+        :noindex:
+
+        Add a current stimulus at each location in ``locations``.
+        Returns a unique identifier that can be used to query the local indexes (see :gen:`index`) assigned to the placed items on the cable cell.
+
+        :param str locations: description of the locset.
+        :param stim: the current stim.
+        :type stim: :py:class:`i_clamp`
+        :rtype: int
+
+    .. method:: place(locations, d)
+        :noindex:
+
+        Add a voltage spike detector at each location in ``locations``.
+        Returns a unique identifier that can be used to query the local indexes (see :gen:`index`) assigned to the placed items on the cable cell.
+
+        :param str locations: description of the locset.
+        :param d: description of the detector.
+        :type d: :py:class:`threshold_detector`
+        :rtype: int
+
+    .. method:: discretization(policy)
+
+        Set the cv_policy used to discretise the cell into control volumes for simulation.
+
+        :param policy: The cv_policy.
+        :type policy: :py:class:`cv_policy`
+
+.. py:class:: cable_cell
```

Comment:
Shouldn't the `cable_cell` class API be in `python/cable_cell.rst`? I think there's some duplication going on.

Reply:
And resolved!